scripts/deploy.py

```diff
@@ -0,0 +1,19 @@
+# import my contract
+from brownie import MyToken, LinearVesting, accounts
+
+
+def main():
+    account = accounts[0]
+
+    # Deploy the token
+    initialSupply = 1000000 * 10**18
+    myToken = MyToken.deploy(initialSupply, {"from": account})
+    print("Token:", myToken.address)
+
+    # Deploy the vesting contract
+    duration = 180
+    cliff = 90  # test cliff
```

Comment:
Define settings in a dictionnary prior to any deployment, to ease the modification at a single place in the code